packages/common/client-utils/tsconfig.json

```diff
@@ -1,13 +1,10 @@
 {
-	"extends": [
-		"../../../common/build/build-common/tsconfig.base.json",
-		"../../../common/build/build-common/tsconfig.cjs.json",
-	],
-	"include": ["src/**/*"],
-	"exclude": ["src/test/**/*"],
+	"extends": ["../../../common/build/build-common/tsconfig.node16.json"],
 	"compilerOptions": {
 		"rootDir": "./src",
-		"outDir": "./dist",
+		"outDir": "./lib",
 		"baseUrl": ".",
 	},
+	"include": ["src/**/*"],
+	"exclude": ["src/test/**/*"],
```

Comment:
FYI - I after looking at the results, I decided not to automatically move include/exclude to the end in the conversion script.

Most of our tsconfigs have these at the top, which feels most natural.